ROOTFS/etc/waggle/sanity/fatal/core_nvme_emmc.test

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# nx core tests
+pf="NVME and eMMC tests"
+
+echo "$pf: Begin"
+
+# properly mounted tests 
+if [ "$(df -h | grep "/media/root-ro")" = "" ];
+then
+        echo "$pf: RO partition not mounted FAIL"
+        exit 1
+fi
+
+if [ "$(df -h | grep "/media/root-rw")" = "" ];
+then
+        echo "$pf: RW partition not mounted FAIL"
+        exit 2
+fi
+
+if [ "$(df -h | grep "/media/plugin-data")" = "" ];
+then
+        echo "$pf: plugin-data not mounted FAIL"
+        exit 3
+fi
+
+# fsck pass
+if [ "$(cat /run/initramfs/fsck.log | grep "/dev/mmcblk0p1: clean")" = "" ];
+then
+        echo "$pf: incorrect fsck pass on nx-core FAIL"
+        exit 4
+fi
+
+# check for swaps
+if [ "$(swapon | grep "/dev/nvme0n1p1")" = "" ];
+then
+        echo "$pf: swap not properly mounted FAIL"
+        exit 5
+fi
+
+# checking initrd test
+if [ "$(file -L /boot/initrd | grep "gzip compressed data")" = "" ];
+then 
+        echo "$pf: initrd is not a valid archive FAIL"
+        exit 6
+fi
+
```

Comment:
Can you add tests for the Kernel (/boot/Image, /boot/Image.sig) and device tree (/boot/dtb/tegra194-xavier-nx-cti-NGX003-WAGGLE-WS.dtb, /boot/dtb/tegra194-xavier-nx-cti-NGX003-WAGGLE-WS.dtb.sig)